XTS/localjunit/crash-recovery-tests/src/main/java/com/arjuna/qa/extension/BaseServerKillProcessor.java

```diff
@@ -122,7 +122,9 @@ protected void dumpProcesses(Container container) throws Exception {
         String scriptName = config.get("scriptName");
 
         String dir = "target" + File.separator + "surefire-reports" + File.separator + "processes";
-        runShellCommand("mkdir " + dir);
+        if(!new File(dir).exists()) {
+            runShellCommand("mkdir " + dir);
```

Comment:
What if the second run of the command also fails? How about just throw an exception?